tutorials/deploy-ubuntu-20-04-instance-scaleway-elements/index.mdx

```diff
@@ -0,0 +1,127 @@
+---
+meta:
+  title: How to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements
+  description: In this tutorial you will learn how to deploy, upadate and manage a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements.
+content:
+  h1: How to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements
+  paragraph: In this tutorial you will learn how to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway Elements.
+tags: Ubuntu instance setup server 
+  - compute
+dates:
+  validation: 2021-12-02
+  posted: 2021-12-02
+---
+
+[Ubuntu](https://ubuntu.com/) is one of the most popular Linux distributions. 
+First released in 2004, Ubuntu whose name is based on an ancient African word meaning ‘humanity to others’, became quickly the favourite distributon for users around the world, mostly because it is easy to install and use. 
```

Comment:
```suggestion
First released in 2004, Ubuntu whose name is based on an ancient African word meaning ‘humanity to others’, became quickly the favourite Linux distributon of users around the world, mostly because it is easy to install and use. 
```